### ISSUE TITLE
Почему приложение удалено с Google Play?

### ISSUE THREAD

**Issue by Quireg** [vortexwolf/2ch-Browser#109]
@vortexwolf , нарушает ли оно какие нормы?

**Comment by Quireg** [comment]
@vortexwolf призываю тебя. Постучись мне в скайп пожалуйста, quireg.

**Comment by vortexwolf** [comment]
Удалено из-за случайных картинок голых баб. Нормального способа исправить я не нашел. Наверно можно было бы удалить список досок совсем и оставить только строку для ввода адреса, но это сложно и вряд ли поможет.

**Comment by Quireg** [comment]
@vortexwolf я могу опубликовать приложение с соответствующими исправлениями?

**Comment by vortexwolf** [comment]
Да, если знаешь что сделать чтобы не удаляли, также нужно будет поменять namespace, т.к. прошлые уже нельзя использовать.

**Comment by ololoe** [comment]
>Удалено из-за случайных картинок голых баб. 

Штоблядь? Каких баб? На дваче которые? Может тогда браузеры запретить, потому что ими цопе качать можно?

**Comment by Quireg** [comment]
@vortexwolf дай мне ключ которым ты подписывал что бы  я мог выкапывать билды не требующие  перестановки приложения.

**Comment by vortexwolf** [comment]
@Quireg напиши свой адрес здесь или напиши запрос на почту vortexwolf@gmail.com, я в письме пришлю
